azure-pipelines.yml

```diff
@@ -1,41 +1,161 @@
-#
-# See https://docs.microsoft.com/en-us/vsts/pipelines/yaml-schema for details
-#
+variables:
+  Build.Repository.Clean: true
+  _TeamName: DotNetCore
+
+  # Variables for public PR builds
+  ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest'), contains(variables['Build.DefinitionName'], 'github')) }}:
+    _HelixType: build/product
+    _HelixSource: pr/aspnet/AspNetCore-Tooling/$(Build.SourceBranch)
+    # These are needed to suppress a warning in the cibuild invocation since AzDO leaves the `$(_SignArgs)` in place and it fails to resolve.
+    _SignArgs: ''
+    _OfficialBuildIdArgs: ''
+    _PublishArgs: ''
+
+  # Variables for internal Official builds
+  ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest'), contains(variables['Build.DefinitionName'], 'official')) }}:
+    _HelixType: build/product
+    _HelixSource: official/aspnet/AspNetCore-Tooling/$(Build.SourceBranch)
+    _SignType: test
+    _SignArgs: /p:DotNetSignType=$(_SignType) /p:TeamName=$(_TeamName)
+    _OfficialBuildIdArgs: /p:OfficialBuildId=$(Build.BuildNumber)
+    _PublishArgs: '/p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat) /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)'
+
```

Comment:
I've revised a bunch of stuff in the AspNetCore-Tooling repo since you took a copy, mostly to enable asset publishing. You might want to resolve that now, and just set BAR publishing to false